### PR TITLE
@mzikherman: Normalize image responses

### PR DESCRIPTION
**Pull request by dzucconi**
Fun times

### CODE REVIEW

**Comment by dzucconi** [comment]
With this change https://github.com/artsy/force/pull/4101 isn't necessary and that page would have rendered fine. No more `replace` errors as this removes them from the response entirely

File: schema/image/normalize.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import {
```
Comment (mzikherman) [inline schema/image/normalize.js]: oo kinda like this, selecting just what you need rather than the whole lib

File: schema/image/normalize.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  assign,
+  compact,
+  flow,
+  includes,
+  last,
+  isArray,
+  isString,
+} from 'lodash';
+
+export const grab = flow(pick, values, first);
```
Comment (mzikherman) [inline schema/image/normalize.js]: omg so functional

File: schema/image/normalize.js (hunk 1 above)
Comment (dzucconi) [inline schema/image/normalize.js]: Yeah, I dunno; i like the way the resulting code reads, but this big import statement is ugly. Though we do this for GraphQL too...

**Comment by mzikherman** [comment]
:clap: :balloon:

**Comment by mzikherman** [comment]
Very nice.

**Pull request merged**